docs/Change_Log.md

```diff
@@ -1,5 +1,37 @@
 # Change Log
 
+## User-Installed Apps General Availability
+
+#### June 27, 2024
+
+> danger
+> This entry includes breaking changes
+
+Back in March, we announced [the beta for user-installed apps](#DOCS_CHANGE_LOG/userinstallable-apps-preview). After listening and making updates based on feedback from developers and modmins, we're excited to announce that user-installed apps are now considered generally available and can be used in all servers (regardless of size).
+
+With this update, there are a few API and behavioral updates for user-installed apps.
+
+###### API Updates
+
+- `user_id` has been removed from the `interaction_metadata` field on messages. Instead, you can use the `id` field in the nested `user` object. See the [Message Interaction Metadata Object](#DOCS_RESOURCES_CHANNEL/message-interaction-metadata-object) for details.
+- User-installed apps are now limited to creating a maximum of 5 [follow-ups](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/followup-messages) when responding to interactions. This only affects the [Create Followup Message endpoint](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/create-followup-message), and apps installed to the server are unaffected.
+- On [Interactions](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-structure), the value of `authorizing_integration_owners` is now correctly serialized as a string. . Previously, the `"0"` value was incorrectly serialized as a number.
+- `app_permissions` on [Interactions](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-structure) now correctly represents the permissions for user-installed apps. Previously, the value was incorrect for user-installed apps.
+- Updating a message can result in a `400` response if the content of the message was blocked by AutoMod, which may be particularly important for [deferred messages](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/responding-to-an-interaction).
+- Interaction responses are no longer forced to be ephemeral for servers with over 25 members.
+
+###### New `Use External Apps` Permission
+
+A new [`USE_EXTERNAL_APPS` (`1 << 50`) permission](#DOCS_TOPICS_PERMISSIONS/permissions-bitwise-permission-flags) was added, and is disabled for servers by default. The new permission lets modmins control whether user-installed apps can post public replies in a server. If `Use External Apps` is disabled and your app is *not* installed to the server, your app’s responses will be ephemeral for the end user.
```

Comment:
what is the motivation behind disabling this permission by default? i assume the vast majority of server owners just leaves the default settings and thus will never enable this feature even if they would like it, which would very negatively affect adoption of this feature 😟 

please reconsider. why not instead add a moderatr-only ephemeral note to user-app responses? it could inform them what this is and how to disable it if they dislike it.

Reply:
oops i had this backwards in my head—user-installed apps def enabled by default and will update this